.changeset/angry-shirts-explode.md

```diff
@@ -0,0 +1,5 @@
+---
+"ledger-live-desktop": major
```

Comment:
Are you sure about the `major` ?

Reply:
indeed this should not bump the `major` version number of LLD, it's rather a patch 👍